pkg/privilege/privileges/cache.go

```diff
@@ -1863,13 +1945,32 @@ func (h *Handle) Get() *MySQLPrivilege {
 }
 
 // Update loads all the privilege info from kv storage.
-func (h *Handle) Update() error {
-	var priv MySQLPrivilege
-	err := priv.LoadAll(h.sctx)
+func (h *Handle) Update(userList []string) error {
```

Comment:
please split into `UpdateAll` and `Update`, a nil `userList []string` is not a clear parameter to mean we want to update all

Reply:
Done